### ISSUE TITLE
Welcome page for a better new user experience

### ISSUE THREAD

**Issue by mikeal** [node-forward/welcome#1]
A single "welcome" page. There are are 4 sections, each of which are very inviting, intended for people: new to programming, new to javascript, new to node, ready to dive deeper in to node.

In the copy of each section we can tailor the language to fit the experience level of these users and, for now, forward them on to resources that are well tailored to their skill level. If we can't find acceptable resources, or we think we can do better, we can spin up specific projects that help people in that skill range.

Should we just add a, index.html to this repo and go from there?

People who I know want to pitch in:

@adambrault
@indexzero
@hackygolucky
@zwigby
@fiveisprime
@polotek
@diasdavid
@olizilla
@hughsk
@iancrowther
@jedwatson

**Comment by hackygolucky** [comment]
Is this where the conversation should be happening? I'm finally catching my breath and wondering how/where I should be giving input :D

**Comment by daviddias** [comment]
@hackygolucky I believe there isn't one yet, but we definitely should talk, hopefully in some kind of video/face to face meeting so we all get into the same page :)

**Comment by hackygolucky** [comment]
@diasdavid sounds great! I'd love to get a call scheduled for **all of us** to lay out priorities, what needs to happen to make these priorities actionable, and areas of responsibility people would like to own. We'll build an agenda so everyone knows what to expect from the meeting and we don't get _too_ far off topic.

I personally would like to focus on building a diverse community of programmers(focusing on underrespresented groups both as adults and kids), and my belief is that in order to do that we need to address newer programmers on top of new-to-node. This approaches the notion that we build the community we want to see by creating AND supporting the programmers we want here. From my conversations with other brave souls in other languages that have attempted to do this, workshops and online docs/tutorials that start from absolute newb have been the most successful--traction, returning interest, excitement. This is a huge responsibility to bite off, as that means we are having to educate from the ground up and making sure we have a 'Start from ground-zero'. I'd love to pull @jasonrhodes in on this repo and gain some more perspective, as he was an incredible energy with Nodeschool in Baltimore and was trying to build a JS 101 Workshopper to help with this very problem. 

It sounds like the direction this is heading is what @mikeal and I had talked about when a few of us whiteboarded with nodejs.org at KnodeCamp(see [knode.io](http://knode.io/) for what the heck it is/was) back in March, which is where we were initially talking on the necessity of creating a real new user experience that worked.

**Main(initial view)**
- knowledge/docs/tutorials
- meetups
- cfp for upcoming events
- Node.js news aggregator
  **note that website should be SUPER simple to contribute to

**Welcome/Getting-Started-with-Node**
- Node.js resources(curated list of how-to’s--Raquel had a great start with her crowd-sourced spreadsheet [HERE](https://docs.google.com/spreadsheet/ccc?key=0Ai2sCp3HpDyGdDRVZWJmaGItaU5BY0NDWEFfY3cyU1E&usp=drive_web#gid=0) 
- JS resources(ex: JS for cats, JS 101 workshopper)
- link to ‘mentors’
- nodejs.org API
- npm(how to npm too)
- #nodejs freenode(and other friendly and accessible channels)
- ‘find your local group’ search

**Mentors**
This was to be a section spearheaded by @rosskukulinski and @groundwater that involved a much more in-depth take on how we build a network of mentors, how you get to be a mentor, and how mentors/mentees find each other. This type of support is key in keeping our community personable and helping to raise each other up.

**Speakers in Node.js**
- a place that local events/confs can look for speakers(from knode/nodemeatspace)

**Events**
- look at nodemeatspace under knode--community contributes upcoming events to calendar
- have a link to ‘start a meetup’

**Comment by jasonrhodes** [comment]
Hey I am super interested in this topic and would love to be involved in discussions etc, thanks Tracy :)

**Comment by mikeal** [comment]
This just reminded me, I think I forgot one of the most important parts, that we need to not only get people access to the knowledge but also to a support system that they can rely on. A good example of this is the NodeSchool discussions repo on GitHub. By having people use GitHub we're not having them learn a tool that lives in a box dedicated to learning, we're actually onboarding them on to a tool that will help support them indefinitely as programmers.

I'm also +1 on focusing on "new to programming."

**Comment by othiym23** [comment]
If there's a way we can incorporate the nodejs Google Group into this, please let me know and I will involve myself. Right now, the group is indeed mostly a tool for newcomers and the perplexed; not a lot of thought-leading going on there. After moderating it for six months or so, I'm convinced that it does provide enough value to make it worth keeping around, but as a support tool it could definitely be more useful integrated into a larger framework.

**Comment by hackygolucky** [comment]
Sounds great! We can add it to the 'tools in your toolbox' for getting help
with stuff.

On Mon, Aug 25, 2014 at 2:01 PM, Forrest L Norvell <notifications@github.com

**Comment by jasonrhodes** [comment]
@othiym23 For some reason I've never been able to get the hang of using Google Groups at all, but I wouldn't mind poking my head in and responding if you could use a hand with something like that. How do I find it?

**Comment by mikeal** [comment]
+1 on moving away from Google Groups and towards GitHub discussion repo issues :)

**Comment by othiym23** [comment]
Yeah, @mikeal, I know how you feel about the mailing list, but part of providing a supportive ecosystem is spreading a wide net to cover as wide an audience as possible. The mailing list is still useful for a surprisingly large group of people.

@jasonrhodes it's at https://groups.google.com/forum/#!forum/nodejs, and it's always useful to have more people pitching it. I moderate the list, but that's mostly to keep out spam an inappropriate jobs postings.

**Comment by mikeal** [comment]
@othiym23 the problem with the mailing list is that it is _not_ a support structure and lives, more or less, outside of the ecosystem where everyone else finds support.

I'm not saying that "email suck." My main mode of reading threads like this is email notifications, and that is precisely my point, there is very little technical differences between a discussion repo and a Google Group but a world of difference in the access you can get to members of the community simply by @ mentioning them or referring to issues and PRs in other repos.

**Comment by mikeal** [comment]
@othiym23 btw, i appreciate the work you've done to make the mailing list usable for the people who go there. My point is that you've had to spend a lot of extra time curating it _because_ it lives outside of the support structure that works for the rest of the node ecosystem. I would much rather have you doing that on GitHub where you can @ mention people you want to bring in, and they actually reply :)

**Comment by hackygolucky** [comment]
Is the search in GH issues good enough that'd it'd be able to replace the
breadth of info on Google Groups? There's so so much in there.

**Comment by mikeal** [comment]
By "search" do you mean search within GitHub in that repo or do you mean search within a Google Search for general terms?

As far as "within Google search," in my experience it is _as_ good as Google Groups but **not** as good as Stack Overflow.

-Mikeal

**Comment by olizilla** [comment]
@hackygolucky are you up for leading the charge? Let's schedule a call and figure out what the first couple of milestones are. I'm on GMT +1 time.

**Comment by hackygolucky** [comment]
Sure thing, @olizilla!

I'll run with a weekday time and if I get too many 'nays' will happily reschedule. We'll set an agenda prior in a format that's editable by everyone listed. Input before the meeting through that agenda will allow everyone their hw time for their priorities in this project and minimize a marathon meeting(which I'm sure none of us has time for).

**Tuesday, September 9th**
**3:30pm GMT**, so that's 
San Francisco 8:30am
NYC 11:30am
London 4:30pm

**PURPOSE**
 Lay out priorities for New User experience page project, what needs to happen to make these priorities actionable, and areas of responsibility people would like to own. 

If y'all have an email that doesn't match your Github acct to be contacted for rsvp of this meeting, please let me know!

@adambrault
@indexzero
@hackygolucky
@zwigby
@fiveisprime
@polotek
@diasdavid
@olizilla
@hughsk
@iancrowther
@jedwatson
@jasonrhodes
@mikeal 
@othiym23 
@nvcexploder

**Comment by olizilla** [comment]
Aces! I'm a yay for the 9th. Of note, it's nodeconf.eu 7-11th Sept, so a handful of the motely crew may be all up in a castle in Ireland at that point.

**Comment by mikeal** [comment]
ya, i'll at at NodeConf but don't delay it on my account.

**Comment by nvcexploder** [comment]
Me too!!

**Comment by zwigby** [comment]
I'm in for the 9th.

**Comment by nvcexploder** [comment]
We can gather interested parties at Nodeconf EU and join.

**Comment by hackygolucky** [comment]
Sounds great. I forgot nodeconfeu was so soon! Let me know if I should
shift it a week. I will be there in spirit :)

**Comment by JedWatson** [comment]
I'm in for the 9th! It's 1:30am in Sydney, I'll stay up :)

Wish I could be at nodeconfeu, would be great to meet more of you all in person.

**Comment by hackygolucky** [comment]
You're crazy and THANK YOU.

**Comment by hughsk** [comment]
I'm in! GMT+2 so that time works great :)

**Comment by junosuarez** [comment]
I found a new node user at BeerJS last night, let me know if you want me to send him your way for <s>interrogation</s>, er, user research

**Comment by hackygolucky** [comment]
Gonna put this here because I think it's wonderful and time well spent on reading and reflection.
https://speakerdeck.com/evildmp/all-you-need-is-l-star-star-star

**Comment by adamavenir** [comment]
@hackygolucky that's one of the most important talks I've ever read. wow.

**Comment by hackygolucky** [comment]
@adambrault It was a great reminder of why I got so excited when I first tried programming. I was exposed to wonderful people with this same attitude.

**Comment by jasonrhodes** [comment]
Seeing more of these talks surfacing lately, it's awesome and a credit to
the conferences that are giving them the stage, too!

**Comment by hackygolucky** [comment]
Yeah, I agree. I believe this was the keynote at DjangoCon this year. It
was very purposely positioned that way. Jacob Kaplan-Moss has left a great
legacy for how communities should be inclusive. Their sprints at
conferences make a point to recruit new/er programmers to try their hand at
contributing to docs, tiny bits of projects, whatever they can muster up
the courage to try out. I know for PyLadies, this has been a way to
encourage retention when the going gets tough for transitions from beginner
to 'great now where the heck do I go from here?'.

**Comment by Fishrock123** [comment]
We should totally have a collection of friendly splash messages on the main / welcome page that are randomly displayed. ^_^

**Comment by hackygolucky** [comment]
That's a wonderful idea!
